common/views/components/AppContext/AppContext.tsx

```diff
@@ -4,23 +4,29 @@ import {
   useState,
   useEffect,
   ReactElement,
-  FunctionComponent,
+  FC,
   ReactNode,
 } from 'react';
 import theme, { Size } from '../../../views/themes/default';
 
+export type PlaybackRate = 0.5 | 1 | 1.5 | 2;
```

Comment:
it seems like you might save some pain by just having this as a `number`(?)

Reply:
deffo